### DISCUSSION TITLE
userAllowedIp

### DISCUSSION THREAD

**Discussion by Smiley-k** [Ylianst/MeshCentral#6364] [General]
using the latest version 1.1.27 

question is it possible when using userAllowedIp (I specified only necessary internal networks) and published to the street meshcentral - everything works correctly, if the IP address is different from the list in the list of allowed I get a stub Unauthorized. Hence the question, is it possible to make a redirect somewhere that would not be unnecessary load or block the connection (perhaps there are some other options that would redirect access)

**Comment by si458** [comment]
have you checked the schema file?
`ipBlockedUserRedirect - If set, a user from a banned IP address will be redirected to this URL.`
example:
`"ipBlockedUserRedirect": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"`

**Reply by Smiley-k** [reply]
> have you checked the schema file? `ipBlockedUserRedirect - If set, a user from a banned IP address will be redirected to this URL.` example: `"ipBlockedUserRedirect": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"`

Oh, I apologize for my inattention. I honestly read the instructions, but I either missed it or didn't take enough importance to it. This is what me needed, thank you very much.
